test-cases/performance/perf-regression-throughput-baremetal-5gb.yaml

```diff
@@ -0,0 +1,54 @@
+test_duration: 90
+
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=10 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..312500",
+                    "cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=10 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=312501..625000",
+                    "cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=10 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=625001..937500",
+                    "cassandra-stress write no-warmup cl=ALL n=312500 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=10 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=937501..1250000"]
+
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=1250000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=4 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..1250000"
+stress_cmd_r: "cassandra-stress read no-warmup  cl=QUORUM duration=30m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=4 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..1250000,625001,1250000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=30m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=4 -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..1250000,625001,1250000)' "
+stress_multiplier: 1
+instance_type_db: 'i3.2xlarge'
```

Comment:
I think this should be removed, we should have code using it beside provisioning code

Reply:
Will do

Reply:
There's a slight problem with changing this - in the pipeline the SCT Runner step actually runs with `AWS` configuration, to create a runner inside AWS, so I'm not sure if I can remove it (talking about `instance_type_db`)

Reply:
So check, having unrelated configuration would cause confusion